src/EnergyPlus/Plant/Enums.hh

```diff
@@ -121,7 +223,7 @@ enum class FlowMode
     LeavingSetpointModulated,
 };
 
-enum class CondenserType
+enum class nCondenserType
 {
     Unassigned,
```

Comment:
`Unassigned` should be -1. Does it have to be 0? Also why is this called `nCondenserType` and not just `CondenserType`?